testcases/misc_tests/p11sak_test.sh

```diff
@@ -154,6 +164,26 @@ echo "* TESTCASE remove-key 3des FAIL Failed to delete generated 3DES key"
 fi
 
 
+# CK_BBOOL
+if [[ $(grep -A 23 'p11sak-3des' $P11SAK_3DES_LONG | grep -c 'CKA_IBM_RESTRICTABLE: CK_FALSE') == "1" && $rc = 0 ]]; then
```

Comment:
Isn't such a block for checking the attributes missing above for DES ? You do perform the list operations in long form for DES keys as well, but check the output for 3DES, AES, RSA, EC only?

Reply:
Is it guaranteed that all key objects created by this testcase always have the CKA_IBM_RESTRICTABLE attribute? As far as I can tell, only the EP11 token supports this attribute, so keys generated with other tokens will not have that attribute at all.

Reply:
Done